heudiconv/cli/run.py

```diff
@@ -204,8 +209,8 @@ def get_parser():
                         ),
                         help='custom actions to be performed on provided '
                         'files instead of regular operation.')
-    parser.add_argument('-g', '--grouping', default='studyUID',
-                        choices=('studyUID', 'accession_number'),
+    parser.add_argument('-g', '--grouping', default='None',
+                        choices=('studyUID', 'accession_number','None'),
```

Comment:
@leej3 - what do you want to say/achieve when groupping is `'None'`?  do you want
- to convert all provided DICOMs as a part of the single study as if their studyUID was the same?   (then I would call it "all" or something like that)
- to convert every sequence as it was a study on its own? (that is where `None` is making sense to me "semantically")

Also I would prefer to keep `studyUID` as the default for a number of reasons:
- consistent behavior with previous versions
- it is the most typical case AFAIK

Reply:
Yes, default of studyUID sounds good. And agreed, 'all' might be more intuitive. Not sure what I would instinctively interpret 'None' to mean.